client/src/result/completed-result-admin-view.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component, PropTypes } from 'react';
+import { Card, CardTitle, CardText } from 'material-ui/Card';
+
+export default class CompletedIndividualResult extends Component {
+
+  static get propTypes() {
+    return {
+      firstComments: PropTypes.string,
+      firstName: PropTypes.string,
+      firstNameNumber: PropTypes.string,
+      id: PropTypes.number,
+      performanceDate: PropTypes.string,
+      performanceName: PropTypes.string,
+      secondComments: PropTypes.string,
+      secondName: PropTypes.string,
+      secondNameNumber: PropTypes.string,
+      spotId: PropTypes.string,
+      userNameNumber: PropTypes.string.isRequired,
+      winnerId: PropTypes.string
+    };
+  }
+
+  render() {
+    const {
+      firstComments,
+      firstName,
+      firstNameNumber,
+      performanceName,
+      secondComments,
+      secondName,
+      spotId,
+      userNameNumber,
+      winnerId
+    } = this.props;
+    const textDivPadding = '0 16px';
```

Comment:
Let's move away from inline styling. Fun experiment, but no other place does it and at the very least, we should be consistent.